README.md

```diff
@@ -24,11 +24,11 @@ Please read this [blog post](https://tvc-16.science/hacktoberfest-lrr-2.html) fo
  
 |Main Page, Thumbnail View | Main Page, List View |
 |---|---|
-| [![archive_thumb](./tools/_screenshots/archive_thumb.jpg)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_thumb.jpg) | [![archive_list](./tools/_screenshots/archive_list.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_list.png) |
+| [![archive_thumb](./tools/_screenshots/archive_thumb.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_thumb.jpg) | [![archive_list](./tools/_screenshots/archive_list.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_list.png) |
```

Comment:
```suggestion
| [![archive_thumb](./tools/_screenshots/archive_thumb.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_thumb.png) | [![archive_list](./tools/_screenshots/archive_list.png)](https://raw.githubusercontent.com/Difegue/LANraragi/dev/tools/_screenshots/archive_list.png) |
```

Reply:
The archive_list screenshot doesn't show the table taking the full width of the page.  
This is an old bug when switching back from thumbnail view I haven't fixed, but since it goes away on a F5 I'd rather not showcase it in the README.